sdk/metricsadvisor/Azure.AI.MetricsAdvisor/src/Generated/MetricsAdvisorModelFactory.cs

```diff
@@ -25,5 +25,14 @@ public static DataSourceCredentialEntity DataSourceCredentialEntity(string crede
         {
             return new UnknownDataSourceCredential(credentialKind, id, name, description);
         }
+
+        /// <summary> Initializes a new instance of <see cref="Models.DataFeedDimension"/>. </summary>
+        /// <param name="name"> dimension name. </param>
+        /// <param name="displayName"> dimension display name. </param>
+        /// <returns> A new <see cref="Models.DataFeedDimension"/> instance for mocking. </returns>
+        public static DataFeedDimension DataFeedDimension(string name = null, string displayName = null)
```

Comment:
`Dimension.DimensionName` property is read-write in specification, but customized in partial file into read-only. HLC ignores read-write / read-only changes.